Code/Engine/Application.cpp

```diff
@@ -29,10 +29,13 @@
 #include "RenderableObject.h"
 #include "EngineStateManager.h"
 
+#include "JsonManager.h"
+
 sad::Window* sad::Application::s_MainWindow;
 
 sad::Application::Application()
 { 
+	sad::JsonManager::ImportLevel();
```

Comment:
Forgot to mention earlier - does this still work properly if we do it after `ResourceManager::Import` now?

Reply:
???